### DISCUSSION TITLE
Special Characters In Metadata

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#452] [Feature Requests]
Is there a way to make special and accented characters in the metadata, such as ö ä á æ ß ♭ ♯, display correctly?

**Comment by rern** [comment]
A track title after edited with Tag Editor:
![0](https://user-images.githubusercontent.com/10094236/127614114-b31d7f13-8c13-491a-9635-c4d7c899257e.jpg)

**Reply by R101G** [reply]
Sorry rern, I should have said that this applies mainly to classical radio stations. I don't have any local files which are affected.

Examples:

Radio Swiss Classic 
http://stream.srg-ssr.ch/m/rsc_de/aacp_96

Naim Classical 
http://mscp3.live-streams.nl:8250/class-high.aac
